safekeeper/src/timeline.rs

```diff
@@ -569,23 +592,28 @@ impl Timeline {
     /// Take a writing mutual exclusive lock on timeline shared_state.
     ///
     /// Unless allowed_cancelled is true errors out if timeline is cancelled (deleted).
+    /// Unless allow_paused is true errors out if timeline is paused.
```

Comment:
Can this comment explain how allow_paused should be used?  What kinds of writes should set allowed_pause=true?

Reply:
added